common/client/transaction_sender_test.go

```diff
@@ -17,37 +17,65 @@ import (
 )
 
 type sendTxMultiNode struct {
-	*MultiNode[types.ID, SendTxRPCClient[any]]
+	*MultiNode[types.ID, SendTxRPCClient[any, *sendTxResult]]
 }
 
 type sendTxRPC struct {
 	sendTxRun func(args mock.Arguments)
 	sendTxErr error
 }
 
-var _ SendTxRPCClient[any] = (*sendTxRPC)(nil)
+type sendTxResult struct {
```

Comment:
why do we need to define new sendTxResult. Can't we reuse existing one?

Reply:
Since this will be extracted to chainlink-framework, we shouldn't import the local EVM sendTxResult. This is also different from the Solana sendTxResult, which includes a signature field, so we don't want to move this to common. That's why I created the testing implementation separately.